.github/actions/test-deployment/action.yml

```diff
@@ -1,108 +1,39 @@
-name: 'Test Deployment'
-description: 'Test deployment using customer workflow'
+name: 'Test Deployment - DEPRECATED'
```

Comment:
Its not clear why we are marking this action as deprecated instead of removing it. Is it relied on elsewhere, hence needing to keep it?

Wouldn't it be better remove it?

Reply:
sure

Reply:
fixt